{% extends "base.html" %}

{% block breadcrumb_links %}
  {{ super() }}
{% endblock breadcrumb_links %}

  {% block content %}
<h1>Delivering accurate structural bioinformatics to the yeast community with the HHyeast data </h1>

<h3>Welcome to the HHyeast website.</h3>

<form action="" method="post" name="fileform">
    {{ form.hidden_tag() }}
    {{ form.filename }}
    {{ form.display }}
    {{ form.download }}
</form>

<p>This website offers the results of remote homology searches of the entire genome of the model
    budding yeast <i>Saccharomyces cerevisiae</i>. The searches have been carried out using the HHsearch
    package developed by Johanne Soeding and colleagues <a href="#ref1">[1]</a>, a tool available online as
    "HHpred" through the <a href="https://toolkit.tuebingen.mpg.de/#/tools/hhpred">server</a>. </p>

<p>The results show visualisations of the strongest homologies for 100% of 6,713 verified yeast open
    reading frames (ORFs) in three databases:
    <ul>
        <li>PDB (solved structures)</li>
        <li>Pfam (curated protein domain families from the European Bioinformatics Institute)</li>
        <li>the yeast proteome itself</li>
    </ul>
</p>

<p>For each ORF, a summary for each of the databases that has at least ine significant hit will be shown. Links at 
    the bottom allow you to change parameters to more detailed views, so you can investigate hits and adjust parameters 
    in one database, and then use those parameters to produce the summary views for all three databases. 
    Note, only hits matching 30 columns or more are reported. </p>

<h3>Gaps with no identified domains </h3>

<p>HHyeast has discovered some additional hits in the gaps between the domains displayed in this website
    <a href="#ref2">[2]</a>. Although these are not included in the visualisations, the data can be
    obtained from the HHyeast Giyhub site, together with all other data for our yeast genome searches. 
    Please email Tim Levine (tim levine @ucl.ac.uk) for data requests.
</p>

<h3>Downloaded data format </h3>

<p>As well as being able to save images of the domain displays through the controls in each window,
    individual files from which the data has been extracted for the visualisation can be downloaded for each ORF.
    This can be done either when selecting the ORF in this page. This file will indicate which residues and what 
    (predicted) secondary structural elements have been aligned. It will also incude hits of <30 matched columns.</p>

<p>The single downloaded data file containing the whole genome search results for all three databases is 
    a text file with suffix ".hhr", and it can be opened by any text editor (e.g. Mac OS X "TextEdit". Each 
    ".hhr" file contains a table of the top 5000 hits to maximise the reach of the tool, but it only has the 
    details of  the top 500 hits for data handling purposes. If you want details of a hit at position 501 or 
   lower, then repeat the search at <a href="https://toolkit.tuebingen.mpg.de/#/tools/hhpred">HHpred</a>.</p>

<h3>Job submission </h3>

<p>In the box at the top of the page, type a gene identifier, either systematic name or standard name if 
    available. The server will offer valid options. 
    When the "Display plot" button is pressed, typically three panels will be displayed,
    one for each of the databases (PDB, Pfam, Yeast). However, a panel will not be displayed if no hits
    reach the default matching probability threshold for display.
    Each panel can then be examined in detail allowing re-setting of display thresholds for the other
    two panels as well.</p>

<br><br>

<h4>REFERENCES </h4>

<p id="ref1">[1] <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1093/bioinformatics/bti125">"Protein homology
    detection by HMM-HMM comparison"</a>, SÃ¶ding J., Bioinformatics, 2005, updated most recently in
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1016%2Fj.jmb.2017.12.007">"A Completely Reimplemented MPI
        Bioinformatics Toolkit with a New HHpred Server at its Core."</a>, Zimmermann et al., J Mol Biol., 2018. </p>
<p id="ref2">[2] "HHyeast reveals hundreds of new domains in the yeast proteome". Christidi et al., manuscript in
    preparation </p>

{% endblock content %}
